Various improvements (#314)
elan-ev/tobira: scripts/README.md

# Helper scripts for development

This directory contains several scripts and docker-compose files that help with development of Tobira.
None of these files should be used in production.

- `auth-proxy`: an nginx implementing an auth proxy as described in `/docs/auth/all-tobira.md`.

>   Listens on port 3090 and forwards requests to 3080, or for login requests to 3091.

Reviewer: ```suggestion
  Forwards requests to port 3090 to port 3080, except for login requests which go to 3091.
```

:shrug:

Author: "to port .. to port" sounds a bit meh. But my original also sounds weird. Will figure something out.

- `dev-db`: development PostgreSQL DB. Listens on default Postgres port.

- `login-handler`: a dummy login handler (with fixed login data) as described in `/docs/auth/all-tobira.md`.
  Listens on port 3091.

> - `opencast-cors`: an nginx that's a reverse proxy in front of your local opencast, allowing CORS request.

Reviewer: ```suggestion
- `opencast-cors`: an nginx that's a reverse proxy in front of your local Opencast, allowing CORS request.
```
